wca-guidelines.md

```diff
@@ -222,6 +222,8 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
+- H1a+) [ADDITION] The competitors may choose to bring 1 or 2 extra puzzles as backup in case of incidents found during the attempt (e.g. duplicate scrambled puzzles). Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.
+- H1a++) [ADDITION] The extra puzzles should be scrambled together with the intended puzzles but kept away from the competitors untill incidents are found.
```

Comment:
Suggestion:
"The extra puzzles must be scrambled together with the intended ones but kept away from the competitor. Extra puzzles must only be used if any incident is found during the attempt."